scripts/env_matrix.yml

```diff
@@ -3,7 +3,7 @@ CONDA_PY:
   - 34
   - 35
 CONDA_BOOST: "1.60"
-CONDA_R: "3.3.1"
+CONDA_R: "3.3.2"
```

Comment:
example run:

['docker', 'run', '--net', 'host', '-v', '/tmp/tmpddqc6jhs/build_script.bash:/opt/build_script.bash', '-v', '/home/dilmurat/miniconda3/conda-bld:/opt/host-conda-bld', '-v', '/home/dilmurat/workspace/bioconda-recipes/recipes/bioconductor-genomeinfodb:/opt/recipe', '-e', 'CONDA_R=3.3.*', '-e', 'CONDA_PERL=5.22.0', '-e', 'CONDA_PY=27', '-e', 'CONDA_BOOST=1.60', '-e', 'CONDA_NPY=110', '-e', 'CONDA_GSL=1.16', '-e', 'CONDA_GMP=5.1', '-e', 'CONDA_HDF5=1.8.17', '-e', 'CONDA_NCURSES=5.9', 'tmp-bioconda-builder', '/bin/bash', '/opt/build_script.bash']
bla
INFO bioconda_utils.build:build(106): BIOCONDA BUILD SUCCESS /home/dilmurat/miniconda3/conda-bld/linux-64/bioconductor-genomeinfodb-1.8.7-r3.3._0.tar.bz2, CONDA_BOOST=1.60;CONDA_GMP=5.1;CONDA_GSL=1.16;CONDA_HDF5=1.8.17;CONDA_NCURSES=5.9;CONDA_NPY=110;CONDA_PERL=5.22.0;CONDA_PY=27;CONDA_R=3.3.*
INFO bioconda_utils.build:build_recipes(370): BIOCONDA BUILD SUMMARY: successfully built 1 of 1 recipes